config/routes.rb

```diff
@@ -18,4 +18,6 @@
   post "/topic/:topic_slug/:subtopic_slug/email-signup", to: "email_signups#create"
 
   get "/government/organisations/:organisation_id/services-information", to: "services_and_information#index", as: :services_and_information
+
+  get '/alpha-taxonomy/:taxon_base_path', to: 'taxons#show'
```

Comment:
I wanted to avoid doing a wildcard here. However, we will have to do that now that taxon URLs have changed. When that content tagger change is live we can start fixing this. There is a trello card for it here: https://trello.com/c/AgKA9XaM/333-render-taxons-and-their-new-base-paths-in-collections